plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/printing/JKCodeBuilder.kt

```diff
@@ -759,10 +760,22 @@ class JKCodeBuilder(context: NewJ2kConverterContext) {
             renderTokenElement(method.rightParen)
         }
 
+        private fun redundantModifiersForConstructor(constructor: JKConstructor): Boolean {
```

Comment:
Can this be merged into `org.jetbrains.kotlin.nj2k.tree.ModifiersKt#isRedundant`?